src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1001,4 +1001,72 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.ExpressionStatement
         End Function
     End Class
+
+    Partial Class BoundAddRemoveHandlerStatement
+        Implements IEventAssignmentExpression
```

Comment:
No, I meant expression. This node is syntactically a statement, and having it act as a statement seems straightforward. Then, a different node can be introduced to act as the expression, and that node can get all of its behavior from this one.

Analyzers are much more likely to be interested in expressions than in expression statements that wrap them. Walkers, on the other hand, care about regular structure, and the irregularity of having an expression with no statement that wraps it would be annoying.